src/lib/transaction_snark/test/multisig_account/multisig_account.ml

```diff
@@ -33,20 +33,30 @@ let%test_module "multisig_account" =
             let open Checked in
             Checked.List.map ~f:(fun (_, pk') -> neq_pk pk pk') xs
             >>= fun bs ->
-            [%with_label_ __LOC__] (fun () ->
+            with_label __LOC__ (fun () ->
                 Boolean.Assert.all bs >>= fun () -> distinct_public_keys xs )
         | [] ->
             Checked.return ()
 
-      let%snarkydef_ distinct_public_keys x = distinct_public_keys x
+      let distinct_public_keys x =
+        let label =
+          Stdlib.(
+            "distinct_public_keys: " ^ __FILE__ ^ ":" ^ string_of_int __LINE__)
+        in
+        let%bind () = with_label label (fun _ -> Checked.return ()) in
```

Comment:
Same thing. 

However, now I'm wondering why you even need the `let%bind`. You could basically do something like `Format.printf "distinct_public_keys: %s : %d@." __FILE__  __LINE__` at this point.